src/pages/about-us.tsx

```diff
@@ -23,36 +23,30 @@ const AboutUs = (props: PageProps<AboutUsQueryProps>) => {
       />
       <AboutUsPage
         heroImageData={props.data.hero}
-        team={props.data.allContentfulTeamMember.nodes}
+        images={props.data.allContentfulGalleryImage.nodes}
       />
     </>
   );
 };
 
 export default AboutUs;
+
 export const AboutUsPageQuery = graphql`
   query ($language: String!) {
-    hero: file(relativePath: { eq: "office/sy-office-02.jpg" }) {
+    hero: file(relativePath: { eq: "office/sy-office-05.jpg" }) {
```

Comment:
Not related to this code line but I have no other place to put this comment inline: 
What's the file `2.png`? It's all black.

Reply:
I need to remove this file, thank you for the comment. It was used as a static image placeholder to test and visualize the gallery tile layout without having to think about things like cropping and where to get the picture files from.